packages/ubuntu_init/test/init_wizard_test.dart

```diff
@@ -174,16 +176,11 @@ void main() {
       InitStep.locale.route,
       InitStep.keyboard.route,
       InitStep.identity.route,
-      InitStep.ubuntuProOnboarding.route,
-      InitStep.ubuntuPro.route,
-      InitStep.ubuntuProSuccess.route,
```

Comment:
I've made the 'init' test run through all steps instead, so we don't need those here.